### PR TITLE
feat(talks overview): add talks overview page

### PR DESCRIPTION
**Pull request by StephanoLioe**

### CODE REVIEW

**Comment by DaveBitter** [comment]
It looks like the theme value is not properly set for the navigation bar

File: pages/talks.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import Image from '@/components/Image'
+
+export async function getStaticProps() {
+  const talks = await getAllFilesFrontMatter('talks')
+  const authors = await getAuthors(talks)
+
+  return { props: { talks, authors, theme: 'rouge' } }
+}
+
+export default function Articles({ talks, authors }) {
```
Comment (DaveBitter) [inline pages/talks.js]: This is still called `Articles` should be `Talks`

File: pages/talks.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                height={1192}
+                layout="responsive"
+                className="rounded-full"
+                alt=""
+              />
+            </div>
+            <div className="col-span-full md:col-span-5 md:col-start-4 xl:col-span-4 xl:col-start-4">
+              <div className="xl:w-11/12">
+                <p className="mb-4 ">
+                  We have some great experts that can deliver some expert talks at your location!
```
Comment (DaveBitter) [inline pages/talks.js]: For @sanderdejong88 , I really think we should rethink this copy. What's the goal of the Talks page? If it is to get our people to talk at events/clients, we should write some copy better geared to that. For instance, who can they contact?

**Review by DaveBitter — COMMENTED** [review]
Nice job, good to have the talks here. Let's also let @lucienimmink know that this is available (if he doesn't read this comment)

File: pages/talks.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+  return (
+    <>
+      <PageSEO title={`Talks - ${siteMetadata.author}`} description={siteMetadata.description} />
+      <section className={`bg-io_${theme}-500 text-white`}>
+        <div className="container mx-auto pt-8 pb-24 md:pb-32">
+          <div className="grid grid-cols-12">
+            <div className="col-start-1 col-end-12 mb-8 md:col-end-8 md:mt-4 md:mb-10 xl:row-start-1 xl:mt-12 xl:mb-16">
+              <h1 className="text-4xl md:text-5xl xl:text-7xl">
+                Awesome <span className="font-serif">Expert talks</span> to share knowledge
```
Comment (mrtnvh) [inline pages/talks.js]: iO brand guidelines says that the serif font has to be of the weight `light`

Comment (sanderdejong88) [reply pages/talks.js]: Then we have multiple places where this is broken. Lets fix them all at once.

**Review by sanderdejong88 — APPROVED** [review]

**Pull request merged**